osx/makefile

```diff
@@ -21,7 +21,6 @@ dmg: $(DMG)
 $(APP):
 	cd $(topdir) && $(PYTHON) setup.py py2app
 	zip -d $(TMP_APP)/Contents/Resources/lib/python2.7/site-packages.zip 'plover/assets/*'
-	ditto --arch x86_64 $(TMP_APP) $(APP)
```

Comment:
Seems like even with the py2app flag, the DMG size benefits from the ditto stripping. On my machine, the DMG is ~80MB, and after ditto it is 50MB.